{"name": "test_login_with_valid_username_but_invalid_password[9110164834-Shashi123-invalid password]", "status": "passed", "parameters": [{"name": "username", "value": "'9110164834'"}, {"name": "password", "value": "'Shashi123'"}, {"name": "expected", "value": "'invalid password'"}], "start": 1707889983023, "stop": 1707889991038, "uuid": "fc02a59d-d88a-491c-9b6e-bf6eeae6ab48", "historyId": "7d7e4ea55860625c6a62577810760d3d", "testCaseId": "7ab72450d62902b9e15c3aa831844ed6", "fullName": "testCases.test_AmazonTest.TestAmazon#test_login_with_valid_username_but_invalid_password", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('setup', 'log_on_failure')"}, {"name": "parentSuite", "value": "testCases"}, {"name": "suite", "value": "test_AmazonTest"}, {"name": "subSuite", "value": "TestAmazon"}, {"name": "host", "value": "shashi-Aspire-A715-41G"}, {"name": "thread", "value": "27537-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testCases.test_AmazonTest"}]}